playbooks/tmpfs/macos.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: MacOS | set OS specific facts
+  set_fact:
+    tmpfs_volume_name: "AlgoVPN-{{ IP_subject_alt_name }}"
+    tmpfs_volume_path: /Volumes
+
+- name: MacOS | mount a ram disk
+  shell:
+    diskutil info "/{{ tmpfs_volume_path }}/{{ tmpfs_volume_name }}/" ||
+    diskutil erasevolume HFS+ "{{ tmpfs_volume_name }}" $(hdiutil attach -nomount ram://64000)
```

Comment:
Shouldn't we put the full volume path here, as in we do to `diskutil info`?

Reply:
Also: can it be that diskutil won't be in PATH (https://superuser.com/questions/213088/diskutil-command-not-found-in-os-x-terminal)? Maybe we should handle this?

Reply:
> Shouldn't we put the full volume path here, as we do when diskutil info is used?

No, `eraseVolume` requires volume name

> Also: can it be that diskutil won't be in PATH (https://superuser.com/questions/213088/diskutil-command-not-found-in-os-x-terminal)? Maybe we should handle this?

Right. I expanded the command to use the full path instead